addons/account/models/chart_template.py

```diff
@@ -641,8 +646,15 @@ def _setup_utility_bank_accounts(self, template_code, company, template_data):
         for company_attr_name, account in zip(accounts_data.keys(), accounts):
             company[company_attr_name] = account
 
+    def _add_foreign_taxes_data(self, company, countries, data):
+        for country in countries:
+            country_data = self._get_foreign_chart_template_data(country, company, reload=True)
+            for model in ('account.tax', 'account.tax.group'):
+                data[model].update(country_data[model])
+        return data
```

Comment:
biased opinion: if we plan to modify the Data-Structure received as an argument, I think it might be better to not return the DS as it may lead people to think it's a new DS and that we left the argument untouched.
Also, I would prefer a name like `enrich_with_{the_stuff_we_re_adding_to_it}` so it makes it crystal clear that we're modifying the argument.